examples/AffinePlaneCurve.jl

```diff
@@ -203,10 +201,8 @@ function Oscar.isirreducible(C::AffinePlaneCurve)
   end
   if length(C.components) != 1
      return false
-  elseif collect(values(C.components)) != [1]
-     return false
   else
-     return true
+     return all(x -> x == 1, values(C.components))
```

Comment:
Just for my understanding: `length(C.components) == 1` at this point, right?

What type are the values of this dictionary? Integers? Or e.g. polynomials? In the latter case, perhaps this would be better (perhaps this would be better in all cases, dunno):
```suggestion
     return all(isone, values(C.components))
```